docs/mp/oci/01_oci.adoc

```diff
@@ -16,63 +16,76 @@
 
 = Oracle Cloud Infrastructure Integration
 :h1Prefix: MP
 :description: Helidon OCI Integration
-:keywords: oci
-:common-deps-page-prefix-inc: ../../shared/dependencies/common_shared.adoc
+:keywords: oci, cdi
+:cdi-extension-api-url: https://jakarta.ee/specifications/cdi/2.0/cdi-spec-2.0.html#spi
+:jsr330-singleton-api-url: https://jakarta.ee/specifications/dependency-injection/1.0/apidocs/javax/inject/singleton
+:oci-objstore-url: https://docs.cloud.oracle.com/iaas/Content/Object/Concepts/objectstorageoverview.htm
+:oci-javasdk-url: https://docs.cloud.oracle.com/iaas/Content/API/SDKDocs/javasdk.htm
+:oci-cli-url: https://docs.oracle.com/en-us/iaas/tools/oci-cli/latest/
+:oci-objstore-javadoc: https://docs.oracle.com/en-us/iaas/tools/java/latest/com/oracle/bmc/objectstorage/package-summary.html
 :feature-name: OCI Integration
+:common-deps-page-prefix-inc: ../../shared/dependencies/common_shared.adoc
 
-Helidon MP OCI Integration provides easy access to Oracle Cloud Infrastructure.
-
-== Experimental
-
-WARNING: Helidon integration with Oracle Cloud Infrastructure is still experimental and not intended for production use. APIs and features have not yet been fully tested and are subject to change.
+Helidon MP OCI Integration provides easy access to
+Oracle Cloud Infrastructure using the OCI Java SDK.
 
 include::{common-deps-page-prefix-inc}[tag=maven-dependency]
 
 [source,xml]
 ----
 <dependency>
-    <groupId>io.helidon.integrations.oci</groupId>
-    <artifactId>helidon-integrations-oci-cdi</artifactId>
+     <groupId>io.helidon.integrations.oci.sdk</groupId>
+     <artifactId>helidon-integrations-oci-sdk-cdi</artifactId>
 </dependency>
 ----
 
-== General Configuration
+== The Helidon OCI SDK Extension
 
-NOTE: If you follow these instructions on how to
-https://docs.oracle.com/en-us/iaas/Content/API/Concepts/apisigningkey.htm#two[Generate an API Signing Key],
-be advised that Helidon does not currently support passphrase-protected private keys in PKCS#1 format.
-If generating a private key using those instructions, use the _no passphrase_ option.
+When added to your application this link:{cdi-extension-api-url}[CDI portable extension]
+provides support for injecting
+link:{oci-javasdk-url}[Oracle Cloud Infrastructure SDK Clients]
+in your Helidon MicroProfile application.
+The extension also handles authenticating with OCI by automatically picking up OCI
+credentials from your environment.
 
-===  Using Helidon MP Properties Configuration
+== Configuring the Helidon OCI SDK Extension
 
-The first option to configure connection to OCI is to directly specify properties in `microprofile-config.properties` file:
+When you inject an OCI SDK Client object, the Helidon OCI SDK extension
+configures and constructs the object for you. The configuration primarily
+consists of initializing an OCI Authentication Details Provider. By default
```

Comment:
Nit: should be `AuthenticationDetailsProvider`; it's a classname

Reply:
Fixed